experimenter/experimenter/nimbus_ui_new/templates/nimbus_experiments/detail.html

```diff
@@ -216,7 +216,23 @@ <h4>Audience</h4>
             </tr>
             <tr>
               <th>Recipe JSON</th>
-              <td colspan="3">{{ experiment.targeting }}</td>
+              <td colspan="3">
+                <div class="collapse" id="collapseRecipe">
+                  <code>
+                    <pre class="text-monospace" style="white-space: pre-wrap; word-wrap: break-word;">
+                      {{ experiment.recipe_json|linebreaks|linebreaksbr }}
+                    </pre>
+                  </code>
+                </div>
+                <button class="btn btn-outline-primary btn-sm"
+                        type="button"
+                        data-bs-toggle="collapse"
+                        data-bs-target="#collapseRecipe"
+                        aria-expanded="false"
+                        aria-controls="collapseRecipe">
+                  <i class="fa-solid fa-plus"></i> Show Recipe
+                </button>
```

Comment:
I'd suggest putting the button above the div so that the button stays in one place when the text expands/collapses.

Reply:
I know I thought of that, I think it makes sense too but one thing I've realized after talking to shell is we should try to keep the behaviour in general as similar to the existing pages before making major changes otherwise it'll be disorienting for users when we completely redo the UI from the new designs from Tylea, so I'm fighting the urge to improve things as I go.

Reply:
Totally valid, though currently the button is in the upper right and stays there when the recipe expands.
![image](https://github.com/user-attachments/assets/7cb245e4-4f7d-4102-89f3-547490bb2ef4)

Reply:
Though that context is good to know for any future UI reviews I may do, so thanks!